backend/src/card_game/api/helper.clj

```diff
@@ -0,0 +1,47 @@
+(ns card-game.api.helper
+  (:require [card-game.core :as core]
+            [card-game.victory-conditions :as victory-conditions]
+            [card-game.persistence :as persistence]
+            [configs :as configs]))
+
+(defn player-num
+  "Translates a Player id into an internal player representation"
+  [game-state id] (.indexOf (:player-ids game-state) id))
+
+(defn translate-player
+  "Translates an internal player to a useful representation"
```

Comment:
"to a human-readable representation" would be more accurate, I think, might as well change these things while we're at it

Reply:
skipped this one 🙃